src/WebAPI/Controllers/RestaurantController.cs

```diff
@@ -16,15 +16,20 @@ public RestaurantController(IRecommendationRepository recommendationRepository)
             _recommendationRepository = recommendationRepository;
         }
 
+
         [HttpGet]
-        public Task<IEnumerable<Restaurant>> Get()
```

Comment:
Has this already been pushed to azure? The client isn't working now, and I'm suspecting that you still need to keep the old function for when no query params are being sent

Reply:
It hasn't been pushed to master so it shouldn't be in Azure yet

I think it's broken because there's longitude / latitude data now but the Restaurant model in prod hasn't been updated